### PR TITLE
Added a functioning demo of Airspeed Velocity for benchmarking

### PR DESCRIPTION
**Pull request by parthb83**
Added and configured `Airspeed Velocity (asv)` according to the cvxpy repository to benchmark it.

### CODE REVIEW

File: benchmark/cvar_benchmark.py [hunk 1]
```diff
@@ -81,4 +73,7 @@ def get_problem_instance() -> cp.Problem:
             ],
         )
 
-        return problem
+        self.problem = problem
+
+    def time_compile_problem(self):
+        self.problem.get_problem_data(solver=cp.SCS)
```
Comment (phschiele) [inline benchmark/cvar_benchmark.py:82]: Perhaps something like this to make debugging easy.

```suggestion
        self.problem.get_problem_data(solver=cp.SCS)
        

if __name__ == '__main__':
    cvar = CVaRBenchmark()
    cvar.setup()
    cvar.time_compile_problem()        
```

File: benchmark/cvar_benchmark.py [hunk 2]
```diff
@@ -15,16 +15,8 @@
...
-
-class CVaRBenchmark(Benchmark):
-    @staticmethod
-    def name() -> str:
-        return "CVaR"
-
-    @staticmethod
-    def get_problem_instance() -> cp.Problem:
+class CVaRBenchmark():
+    def setup(self):
```
Comment (phschiele) [inline benchmark/cvar_benchmark.py]: Some benchmarks were still failing for me, I suggest we set the time out very high for now.
I recommend using `asv run --quick` for developing.

```suggestion
class CVaRBenchmark():
    timeout = 999

    def setup(self):
```

**Review by phschiele — CHANGES_REQUESTED** [review]
Great setup @parthb83 !
I've added a few comments below. Probably you should remove your results from the results folder, which should be filled via CI eventually. CI is not running as you are a first time-contributor to this repo. I suggest running isort and flake8 locally.
After this PR, that should work out of the box.

**Comment by parthb83** [comment]
> Great setup @parthb83 ! I've added a few comments below. Probably you should remove your results from the results folder, which should be filled via CI eventually. CI is not running as you are a first time-contributor to this repo. I suggest running isort and flake8 locally. After this PR, that should work out of the box.

So I pushed the modifications suggested by you @phschiele. Ran `flake8` and `isort` inside the `benchmark/` folder and on `setup.py` file. I would recommend to exclude `asv.conf.json` from flake8 as it may constitute some problems with its syntax.

**Comment by parthb83** [comment]
We will be doing the development on the forked repository until we have a working configuration of asv for benchmarks.

**Pull request closed without merging**